core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -430,6 +431,7 @@ public PropServerConfiguration(
         this.snapshotInstanceId = getString(properties, env, PropertyKey.CAIRO_SNAPSHOT_INSTANCE_ID, "");
         this.snapshotRecoveryEnabled = getBoolean(properties, env, PropertyKey.CAIRO_SNAPSHOT_RECOVERY_ENABLED, true);
         this.simulateCrashEnabled = getBoolean(properties, env, PropertyKey.CAIRO_SIMULATE_CRASH_ENABLED, false);
+        this.rssMemoryLimit = (long) (getDouble(properties, env, PropertyKey.CAIRO_RSS_MEMORY_LIMIT, -1) * 1024L * 1024L * 1024L);
```

Comment:
the PR description states the default value is 0, but this code looks like `-1`

Reply:
also rounding logic is shady. What if someone specifies 0.5GiB ?

Reply:
Changed description. There is test with value 3.5, calculates correctly

```
Assert.assertEquals((long)(3.5 * 1024L * 1024L * 1024L), configuration.getCairoConfiguration().getRssMemoryLimit());
```